src/ai/backend/client/func/session.py

```diff
@@ -99,6 +132,14 @@ class ComputeSession(BaseFunction):
     domain: str
     group: str
 
+    @property
+    def session_id(self) -> Optional[UUID]:
+        return self.id
+
+    @property
+    def session_name(self) -> Optional[str]:
+        return self.name
+
```

Comment:
If the purpose of creating properties was to prevent modifications to id and name,
please make id and name private fields and align the property names accordingly.